### PR TITLE
Author avatar component with bots and guest support

### PR DESCRIPTION
**Pull request by nickvergessen**

### CODE REVIEW

File: src/components/MessagesList/MessagesGroup/AuthorAvatar.vue [hunk 1]
```diff
@@ -0,0 +1,115 @@
+<!--
+  - @copyright Copyright (c) 2019 Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @author Marco Ambrosini <marcoambrosini@pm.me>
```
Comment (ma12-co) [inline src/components/MessagesList/MessagesGroup/AuthorAvatar.vue]: Joas!

**Comment by nickvergessen** [comment]
To review this check out the difference between the changelog room on master and this one

File: src/components/MessagesList/MessagesGroup/AuthorAvatar.vue (hunk 1 above)
Comment (ma12-co) [reply src/components/MessagesList/MessagesGroup/AuthorAvatar.vue]: I mean here you should put your name @nickvergessen

File: src/assets/variables.scss [hunk 2]
```diff
@@ -60,4 +60,4 @@ $breakpoint-mobile: 1024px;
 $message-width: 600px;
 
 //message utils width
-$message-utils-width: 120px  
+$message-utils-width: 120px
```
Comment (ma12-co) [inline src/assets/variables.scss]:
```suggestion
$message-utils-width: 120px;
```

File: src/components/MessagesList/MessagesGroup/AuthorAvatar.vue (hunk 1 above)
Comment (nickvergessen) [reply src/components/MessagesList/MessagesGroup/AuthorAvatar.vue]: done

**Pull request merged**